### DISCUSSION TITLE
Issue with style file import

### DISCUSSION THREAD

**Discussion by marharytaCh** [goveo/react-international-phone#159] [Q&A] [answered]
Hi, I've installed this lib, version 4.2.5. I followed Getting started instructions, but after adding this line 
`import 'react-international-phone/style.css'`
I have an error Unable to resolve path to module 'react-international-phone/style.css'. there is no such file in node_modules folder of this lib. Can you tell me if I`m doing something wrong?

**Comment by ybrusentsov** [comment, marked as answer]
@marharytaCh
It's not an issue with a library. Seems like you don't have a proper loader for CSS files.
Check a solution like this: https://stackoverflow.com/a/49425633

**Comment by ybrusentsov** [comment]
Duplicate of https://github.com/goveo/react-international-phone/issues/97
